pyout.py

```diff
@@ -154,14 +148,14 @@ def _moveback(self, n):
 
     ## FIXME: This will break with stderr and when the output scrolls.
     ## Maybe we could check term height and repaint?
-    def rewrite(self, label, column, new_value, style=None):
+    def rewrite(self, ids, column, new_value, style=None):
```

Comment:
i guess we could back to this later, when actual usecases start to arrive... what I thought might be better is to have the "ids" be a property of the Tabular, and rewrite even not needing to  be a dedicated another method.  
- `ids` could be a property of Tabular so it would need just to remember those 
- the same `__call__` method, while receiving a new record/row would check if it was seen before (according to ids) and update the entire row (since receiving a full record).
but anyways -- for now it is a non issue. we could get back to it later

Reply:
I see.  I hadn't considered a non-explicit rewrite.

ATM my preference is for an explicit rewrite, but that's
unsurprisingly since I wrote the code :)  And of course that's
uninformed by actual use, so let's revisit this later like you
suggest.